src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,50 @@
 package core.basesyntax;
 
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.util.List;
+
 public class WorkWithFile {
+    private static final String[] CSV_CATEGORIES = new String[]{"buy"};
+    private static final String DELIMITER = ",";
 
     public void getStatistic(String fromFileName, String toFileName) {
+        writeToCsvFile(toFileName, readFromCsvFile(fromFileName));
+    }
+
+    private String readFromCsvFile(String fromFileName) {
+        File fileFrom = new File(fromFileName);
+        List<String> csvValues;
+        int buyTotal = 0;
+        int supplyTotal = 0;
+        try {
+            csvValues = Files.readAllLines(fileFrom.toPath());
+            for (String currentLine : csvValues) {
```

Comment:
More stable will be to call split before if once in case in future you will have categories buy,  buyPack, etc. But for now it's enough